examples/lighting-app/efr32/src/main.cpp

```diff
@@ -1,227 +0,0 @@
-/*
```

Comment:
What happened to this? why are we deleting it? I don't see anything that replaces it

Reply:
I was using main to test, but I only mean to delete it from the the PR, not the repo

Reply:
Well, this file needs to be undeleted somehow :)